packages/terra-date-picker/src/react-datepicker/day.jsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import {
+  getDay,
+  getMonth,
+  getDate,
+  now,
+  isSameDay,
+  isDayDisabled,
+  isDayInRange,
+  getDayOfWeekCode
+} from './date_utils'
+
+export default class Day extends React.Component {
+  static propTypes = {
+    day: PropTypes.object.isRequired,
```

Comment:
Is there existing documentation we can pull forward to use as inline comments for these props?

Reply:
react-datepicker has a ton of them. https://github.com/Hacker0x01/react-datepicker/tree/v0.64.0/docs

I don't want to make them public in our repo to consumers, but I would have no problem adding a react-datepicker-docs directory for our use and reference.

Reply:
I guess I was thinking the Inline docs so when code changes are made later one, we knew the purpose of each prop. I don't think this should be exposed to consumers but it would make code maintenance easier (IMO) later on. If we don't want to do it now, can we log an issue to do this later on?

Reply:
I looked through the docs, and they don't have any descriptions on them theirselves. https://github.com/Hacker0x01/react-datepicker/tree/master/docs.

I'd love to add more, but the docs don't give enough of a description to do so.

Reply:
Let's get an issue logged to add inline doc to the prop-types in a follow up PR.

Reply:
https://github.com/cerner/terra-framework/issues/579